packages/dcos-integration-test/extra/test_service_discovery.py

```diff
@@ -72,7 +74,7 @@ def _service_discovery_test(cluster, docker_network_bridge):
 
     app_definition["constraints"] = [["hostname", "UNIQUE"], ]
 
-    with cluster.marathon_deploy_and_cleanup(app_definition) as service_points:
+    with cluster.marathon().deploy_and_cleanup(app_definition) as service_points:
```

Comment:
I don't see a good reason not to just make this do a `marathon` / take the marathon fixture. the `cluster.slaves` count we can move into the base cluster fixture if it's not there already.